### ISSUE TITLE
Insights V2

### ISSUE THREAD

**Issue by maryliag** [cockroachdb/cockroach#83777]
After the creation of [Insights V1](https://github.com/cockroachdb/cockroach/issues/83774),
add new information about: Failed, Full Scan, High Retries and Slow

Figma: https://www.figma.com/file/xpnRkP9cLXERVceRC4cec4/22.2_SQL-obsrv_query-performance?node-id=1086%3A72736

**Comment by maryliag** [comment]
@kevin-v-ngo did we define what is a slow query here? E.g. anything executing for more than 1 sec?
Also, for retries, should we should anything that had a retry or just above X retries?

**Comment by ericharmeling** [comment]
A couple questions:

1. Is this issue still accurate?

    We have all of the insight types listed in the PR description for the Statement Insights pages, except "Full Scan", which I believe is the same as "Suboptimal Plan" (correct?) 

    We still only have "High Contention" for Transaction Insights, but that is waiting on transaction-level rows in `crdb_internal.cluster_execution_insights`.

2. Is this now a duplicate of https://github.com/cockroachdb/cockroach/issues/86405?
3. Is there a separate issue for transaction-level rows in `crdb_internal.cluster_execution_insights`?

**Comment by maryliag** [comment]
1-  This was focus on statements, so we can close this one as everything here was already addressed.
2 - Not duplicated, since the one you mentioned is focused on txn, and this one is for stmts. 
3 - I don't think we have an issue yet, we should create though.

Closing this issue, since it was addressed already
